backend/routes.py

```diff
@@ -39,7 +49,8 @@ def get_public_agents() -> list[dict]:
 
 
 @pike_router.post("/create_agent/{agentId}")
-def create_agent(agentId: u.UUID, body: gb.AgentConfig) -> dict:
+def create_agent(agentId: u.UUID, 
```

Comment:
nitpik:
just learned a convention that any POST call creates something and GET requests. There is no point in having word create in the route.